src/components/Header/Header.css

```diff
@@ -11,7 +11,8 @@ header.Header {
 }
 
 .LogoWrap {
-  margin: 1rem 0 auto 2rem;
+  padding-top: 6px;
```

Comment:
do not use fixed values.
here's how u should proceed: make `LogoWrap` a `flex` container, give it a `padding: 2rem`, and add an alignment of `align-items: center`

Reply:
on it. thanks @mrwagaba